### DISCUSSION TITLE
Merch

### DISCUSSION THREAD

**Discussion by GarthDB** [DesignSystemsEngineer/designsystemsengineer.github.io#41] [Q&A]
I ordered some stickers. Any brilliant ideas on how to distribute?

We could setup a store to cover shipping and handle managing orders. Drop shipping, on demand printing might be nice in the future. Anyone with experience that they'd like to share?

Also, any other ideas for fun merch?

![image](https://github.com/DesignSystemsEngineer/designsystemsengineer.github.io/assets/125516/bce21cef-97a5-4082-8ef4-d0aa5a76a4e6)

![image](https://github.com/DesignSystemsEngineer/designsystemsengineer.github.io/assets/125516/eebdfdf2-775d-49b7-93fd-25c8029325c3)

**Comment by jnjosh** [comment]
I don't have ideas other than where-do-i-send-the-monies?!?!
